StortSpelProjekt/Engine/src/ECS/Components/Audio2DVoiceComponent.cpp

```diff
@@ -81,3 +83,31 @@ void component::Audio2DVoiceComponent::Stop(const std::wstring& name)
 	m_Voices[name].Stop();
 #endif
 }
+
+void component::Audio2DVoiceComponent::pauseToggleAudio(PauseGame* evnt)
+{
+	if (evnt->isPaused)
+	{
+		for (auto voice : m_Voices)
+		{
+			XAUDIO2_VOICE_STATE test;
+			voice.second.GetSourceVoice()->GetState(&test);
```

Comment:
As before, please change the temporary variable name from test to something more descriptive. voiceState perhaps?